src/item/views/ItemDetail.tsx

```diff
@@ -372,8 +394,9 @@ const ItemDetail: FunctionComponent<ItemDetailProps> = ({
 											<ToggleButton
 												type="tertiary"
 												icon="bookmark"
-												active={false}
+												active={bookmarkViewPlayCounts.isBookmarked}
 												ariaLabel={t('item/views/item___toggle-bladwijzer')}
+												onClick={() => toggleBookmark()}
```

Comment:
`onClick={toggleBookmark}`

Reply:
fixed in: feature/bookmarks-in-workspace